npm run test => error

If there is a configuration file in the parent directory, like this
```js
{
"extends": "eslint:recommended",
"parser": "babel-eslint"
}
```
`npm run test` output errors:
```bash
> node Makefile.js test

Validating Makefile.js
Cannot find module 'babel-eslint'
Error: Cannot find module 'babel-eslint'
    at Object.resolve (/Users/zsd/repo/typescript-eslint-parser/node_modules/eslint/lib/util/module-resolver.js:75:19)
    at Object.load (/Users/zsd/repo/typescript-eslint-parser/node_modules/eslint/lib/config/config-file.js:518:42)
    at loadConfig (/Users/zsd/repo/typescript-eslint-parser/node_modules/eslint/lib/config.js:64:33)
    at getLocalConfig (/Users/zsd/repo/typescript-eslint-parser/node_modules/eslint/lib/config.js:126:23)
    at Config.getConfig (/Users/zsd/repo/typescript-eslint-parser/node_modules/eslint/lib/config.js:227:22)
    at processText (/Users/zsd/repo/typescript-eslint-parser/node_modules/eslint/lib/cli-engine.js:223:27)
    at processFile (/Users/zsd/repo/typescript-eslint-parser/node_modules/eslint/lib/cli-engine.js:298:18)
    at executeOnFile (/Users/zsd/repo/typescript-eslint-parser/node_modules/eslint/lib/cli-engine.js:682:23)
    at /Users/zsd/repo/typescript-eslint-parser/node_modules/eslint/lib/cli-engine.js:719:13
    at Array.forEach (native)
Validating JavaScript files
```
Is this expected? thanks in advance for your kindly response. 😄

--- a/.eslintrc.yml
+++ b/.eslintrc.yml
@@ -1,3 +1,5 @@
+root: true
+
 env:
     node: true
 

